src/plugins/utilities/layout/centered-container.js

```diff
@@ -18,7 +18,7 @@ module.exports = function () {
 
     const newUtilities = {
       // Center an element horizontally.
-      '.centered-container': {
+      '.centered-container, .cc': {
```

Comment:
I can see shortening some class name can be useful since there'll be so many.

Reply:
I find we use this one a lot so it made sense to me.